### ISSUE TITLE
World Border Visibility

### ISSUE THREAD

**Issue by TheoRetisch1** [Puremin0rez/WorldBorder#4]
The Worldborder is no longer visualized by particles. Actually, the border should be shown to the player with flame particles just before he or she meets it ((https://imgur.com/a/yg022Jy). This feature does not work anymore. The Worldborder is just completely invisible.

Paper 1.16.1 - 134
WorldBorder 1.16

**Comment by Puremin0rez** [comment]
I have never seen nor heard of this feature before. I cannot find any code related too it, either.

Are you sure you are not coming from a fork of WorldBorder that had some extra feature or something? I am convinced this is not an actual feature in BrettFlans WorldBorder, which is what mine is based on. (the original)

If you really want it to be visualized, I recommend using the [vanilla worldborder command](https://minecraft.gamepedia.com/World_border) to set the barrier up to get the blue wall. You would run `/minecraft:worldborder set radiushere` in the world you want it in. The important part is that you will need to multiply the radius used in the vanilla worldborder command by 2 for it to equal the same radius as the plugin based worldborder radius.

I'll add it to my suggestions list to add a feature in WorldBorder that will automatically visualize the borders using the vanilla blue walls. However, it will only work with square borders, as circles do not support such a feature.

Please let me know if you have any additional information about this particle effect, since I'm stumped on where this feature could be coming from and why it'd be broken.

**Comment by TheoRetisch1** [comment]
Uhh thanks for the detailed answer! I think I've made a mistake. If there was no Worldborder support for it, it was probably simply visualized by Selection Visualizer from Zrips (https://www.spigotmc.org/resources/selection-visualizer.22631/) and the function is now broken. I already experimented with the default worldborder, but at the moment we are using round worlds and actually I think the shape is pretty cool xD. Selection Visualizer also supports round shapes, I'll try to ask for a fix and link your fork :)
